yaml-templates/generic-analysis-properties-file-sonarcloud.yml

```diff
@@ -0,0 +1,42 @@
+# Generic Analysis with SonarCloud - using an existing properties file
+# * If necessary, create an endpoint to connect to SonarCloud
+#
+# * Set the sonarCloudXXX variables below
+# 
+# * Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/windows/dot-net
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  sonarCloudEndpointName: 'SC'
+  sonarCloudOrganization: 'my-org'
+  sonarCloudProjectKey: 'my-projectkey1'
+  sonarCloudProjectName: 'My Project'
+
+steps:
+- task: SonarCloudPrepare@1
+  inputs:
+    SonarCloud: '$(sonarCloudEndpointName)'
+    organization: '$(sonarCloudOrganization)'
+    scannerMode: 'CLI'
+    configMode: 'file'
+    configFile: 'sonar-project.properties'
```

Comment:
It's not obvious where the 'sonar-project.properties' file should be located. Is it checked in to the root of the repo?
Probably worth adding a comment to say
a) where the file should be located, and 
b) possibly also link to the docs for the format of the file e.g. `For more information on the format of the file see here: https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/`

Reply:
Yep, i clarified this.